src/Microsoft.DotNet.Configurer/NuGetCacheSentinel.cs

```diff
@@ -23,7 +23,7 @@ private string NuGetCachePath
             {
                 if (string.IsNullOrEmpty(_nugetCachePath))
                 {
-                    _nugetCachePath = PackageDependencyProvider.ResolvePackagesPath(null, null);
+                    _nugetCachePath = NuGetPathContext.Create(new NullSettings()).UserPackageFolder;
```

Comment:
The lock file will now contain the package folder paths under ``packageFolders``, you shouldn't need to use NuGetPathContext. For any scenario that does need to load the settings NuGetPathContext.Create(*path to project.json directory*) should be used.